docs/user-guide/first-login.md

```diff
@@ -1,7 +1,7 @@
-When logging in to Gramps Web for the first time, a form will be displayed requiring the user to select a language for the frontend and a home person.
+When logging in to Gramps Web for the first time, a Preferences form (labeled User Settings) will be displayed requiring the user to select a language for the frontend and a [Home Person](https://www.gramps-project.org/wiki/index.php/Gramps_Glossary#home_person). The Home Person selection can also be changed from the Home Page of the website.
 
-This home person does not have to coincide with the home person in Gramps desktop &ndash; it is a personal setting for every user of Gramps Web. The home person is used e.g. for the initial view of the family tree. A typical choice would be for a user to select themselves as home person.
+The Home Person for a visitor to the website does not have to be the same person that the site's owner chose in Gramps for Desktops — it is a personal setting that can be different for every user of Gramps Web. The home person is used for the initial view of the family tree and the starting point for most Reports. A typical choice would be for a user to select themselves as Home Person. When collaborating with another member of a Tree, those visitors might want to set the common ancestor as the Home Person. This ensures a common point of reference for the distant relationships.
```

Comment:
I would not use the would "visitor to the website" because it needs to be somebody with a user account, i.e. a user.